ts/sagas/contentLoaders.ts

```diff
@@ -188,7 +188,7 @@ export function* watchContentSaga() {
   // Load content related to the contextual help body
   yield put(loadContextualHelpData.request());
 
-  if (bonusVacanzeEnabled || bpdEnabled || cgnEnabled) {
+  if (bonusVacanzeEnabled || bpdEnabled) {
```

Comment:
Since now `cgnEnabled` is always `true`, I think we sould remove the if condition and always `takeLatest` `loadAvailableBonuses.request` otherwise when `bonusVacanzeEnabled` and `bpdEnabled` are removed from the codebase the action `loadAvailableBonuses.request` will never be taken.

Reply:
You're actually right but I guess we should approach this as part of refactoring of the bonus store section within the app. We should move this from bonusVacanze code base and make it as common without protecting it with FF, what do you think?

Reply:
I totally agree with the bonus refactoring, but I think in this phase could be worth just removing the if and keep the:
```typescript
yield takeLatest(
      getType(loadAvailableBonuses.request),
      handleLoadAvailableBonus,
      contentClient.getBonusAvailable
    );
```
because at the moment is like we have an hidden condition `if (bonusVacanzeEnabled || bpdEnabled || true)` where `true` is the CGN bonus. If we remove both `bonusVacanzeEnabled` and `bpdEnabled` this part could be deemed no longer necessary and removed, while cgn makes use of it to allow registration. You think this could prevent some future misunderestanding?

Reply:
Fixed through 16352a5